### DISCUSSION TITLE
How viable would it be to add Taskbar Media Controls for osu?

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#25647] [Ideas] [answered]
![image](https://github.com/ppy/osu/assets/82095491/ef06becf-5ebc-4acb-9715-65f6390eef15)
I'm not sure if the work needed to implement is worth it. You would need to either disable the buttons or make them do different things when the game is in different places (like editor, gameplay, etc.).
But if people are using osu! as their music player, this would be a good feature. Any opinions?

**Comment by bdach** [comment, marked as answer]
Not happening, osu! is not a music player.
